css/properties/max-height.json

```diff
@@ -93,11 +93,11 @@
               },
               "safari": {
                 "version_added": "9",
-                "notes": "Safari implements an earlier proposal for setting height to an intrinsic height: the keywords <code>intrinsic</code> (instead of <code>max-content</code>), and <code>min-intrinsic</code> (instead of <code>min-content</code>). There is no equivalent for <code>fill-available</code> or <code>fit-content</code>."
+                "notes": "Safari implements an earlier proposal for setting height to an intrinsic height: the keywords <code>intrinsic</code> (instead of <code>max-content</code>), and <code>min-intrinsic</code> (instead of <code>min-content</code>). There is no equivalent for <code>stretch</code> or <code>fit-content</code>."
```

Comment:
The descriptions for Safari are inconsistent across files. `max-height` and `max-width` say Safari 9 supports earlier proposal, `min-height` and `width` say Safari 9 supports `-webkit-fill-available`, and `min-width` says no support.